source/hardware/controllers/denon_mc7000.rst

```diff
@@ -79,13 +79,13 @@ Please check the \*.js mapping file for user variables to:
 
 -  activate NeedleDrop sensor while a track is playing (default: false)
 -  set the Pitch Fader ranges in % to toggle between them (default: 4,
-   6, 10, 16, 24)
+   6, 8, 10, 16, 24)
 -  Platter Ring LED mode: single LED on or off (default: 1). Can be
    switched with SHIFT + Deck button
 -  Vinyl Mode on or off at MIXXX start which also triggers the Platter
```

Comment:
```suggestion
-  Vinyl Mode: on or off at Mixxx start which also triggers the Platter
```
We don't use all uppercase MIXXX, please also fix the other occurrences in this file.

Reply:
done